carbonmark/components/ViewWalletButton/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { cx } from "@emotion/css";
+import { Launch } from "@mui/icons-material";
+import { FC } from "react";
+
+import { ButtonPrimary } from "@klimadao/lib/components";
+import { ButtonBaseProps } from "@klimadao/lib/components/Buttons/ButtonBase";
+import { concatAddress } from "@klimadao/lib/utils";
+import { Trans } from "@lingui/macro";
+import * as styles from "./styles";
+
+type Props = {
+  address: string;
+} & ButtonBaseProps;
+
+export const ViewWalletButton: FC<Props> = (props) => {
+  const className = cx(styles.viewWalletButton, props.className);
+
+  return (
+    <div className={className}>
+      <span>{concatAddress(props.address)}</span>
+      <ButtonPrimary
```

Comment:
Nit: can remove the closing `</ButtonPrimary>` tag and close off the opening tag:

```
<ButtonPrimary  
  {...props}
/>
```

Reply:
done, thanks!